.bazelignore

```diff
@@ -0,0 +1 @@
+c/third_party
```

Comment:
Why is this needed?

Reply:
This avoids, that `BUILD` files in `c/third_party` are directly used in the build graph. However, those folder shall be only used as `workspace`s or `module`s. `.bazelignore` files exist also in most other repos.